lsp/nls/src/linearization/mod.rs

```diff
@@ -24,7 +24,7 @@ pub mod building;
 pub mod completed;
 pub mod interface;
 
-pub type Environment = nickel_lang::environment::Environment<Ident, usize>;
+pub type Environment = nickel_lang::environment::Environment<(Ident, FileId), usize>;
```

Comment:
I don't remember the exact type, but I think we have an alias for items ID?

```suggestion
pub type Environment = nickel_lang::environment::Environment<(Ident, FileId), ItemID>;
```